src/Parser/Priest/Shadow/Modules/Spells/Voidform.js

```diff
@@ -40,136 +36,158 @@ class Voidform extends Analyzer {
     return Object.keys(this._voidforms).map(key => this._voidforms[key]);
   }
 
+  get nonExcludedVoidforms(){
+    return this.voidforms.filter(voidform => !voidform.excluded);
+  }
+
+  get averageVoidformStacks() {
+    if (this.voidforms.length === 0) return 0;
+    // ignores last voidform if seen as skewing
+    return this.nonExcludedVoidforms.reduce((p, c) => p += c.stacks.length, 0) / this.nonExcludedVoidforms.length;
```

Comment:
I recommend breaking this up into a totalVoidFormStacks (which could be another getter to make each block smaller). Also since we're already in the "Voidform" class, I recommend omitting the "VoidForm" from the property name; `totalStacks` would suffice.